<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <item row="1" column="4">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>ch3</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>ch2</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="12">
    <widget class="QCheckBox" name="checkBox_11">
     <property name="text">
      <string>z</string>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QCheckBox" name="checkBox_8">
     <property name="text">
      <string>ch8</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QCheckBox" name="checkBox_7">
     <property name="text">
      <string>ch7</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="11">
    <widget class="QCheckBox" name="checkBox_10">
     <property name="text">
      <string>y</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QCheckBox" name="checkBox_6">
     <property name="text">
      <string>ch6</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QCheckBox" name="checkBox_4">
     <property name="text">
      <string>ch4</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>ch1</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="14">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="13">
    <widget class="QCheckBox" name="checkBox_12">
     <property name="text">
      <string>t</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="14">
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
   <item row="1" column="6">
    <widget class="QCheckBox" name="checkBox_5">
     <property name="text">
      <string>ch5</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QCheckBox" name="checkBox_9">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
